### ISSUE TITLE
Provide browser-specific help with audio/video device settings

### ISSUE THREAD

**Issue by jospoortvliet** [nextcloud/spreed#9898]
Something I heard from partners, about Talk. They weren't using Talk for calls with prospects, because they had to help them often with using Talk. The prospects, often not very technical, would click away the permissions dialog and where some other solutions then show screenshots or an image or arrow on with where to click to give permission, depending on the browser, Talk does not - leaving them to ask what browser is being used and then guide the user through chat to give permission. We can do better, detecting the browser and showing the user how to do it.

This is probably a number of separate improvements - just providing this here as @jancborchardt asked me to.

**Comment by nickvergessen** [comment]
Maybe we could just force the device checker when permissions are not granted and have the details there

**Comment by jancborchardt** [comment]
@jospoortvliet do you think @nickvergessen’s proposal would help already as a quick fix?

**Comment by jospoortvliet** [comment]
Yeah, that would be very nice! I think it would address the issue - ppl can then pick devices, and see that they denied access. Will the request for access pop up again? I guess not, so we'll have to have some way of pointing that out, but that can be there.
